sink/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
     implementation project(':common')
 
 	testImplementation("junit:junit")
+	testImplementation("org.awaitility:awaitility:4.2.0")
```

Comment:
Without this change we can use `4.0.1`. But only starting from `4.0.2` we can use the following (to test that something is true during given period):
```
Awaitility.await().during(20, TimeUnit.SECONDS)
```